src/docfx.website.themes/common/common.js

```diff
@@ -64,34 +64,34 @@ function isRelativePath(path) {
 var gitUrlPatternItems = {
     'github': {
         'testRegex': /^(https?:\/\/)?(\S+\@)?(\S+\.)?github\.com(\/|:).*/i,
-        'generateUrl': function(repo, branch, path, startLine) {
-            var url = normalizeGitUrlToHttps(repo);
-            url += '/blob' + '/' + branch + '/' + path;
-            if (startLine && startLine > 0) {
-                url += '/#L' + startLine;
+        'generateUrl': function (modelGitInfo) {
+            var url = normalizeGitUrlToHttps(modelGitInfo.repo);
```

Comment:
yep, the caller could guarantee it.

---
In reply to: [92736776](https://github.com/dotnet/docfx/pull/979#discussion_r92736776) [](ancestors = 92736776)